packages/cxl-ui/scss/global/cxl-adeft-accordion.scss

```diff
@@ -0,0 +1,55 @@
+@use "~@conversionxl/cxl-lumo-styles/scss/mq";
+@use "../mixins";
+
+cxl-adeft-accordion {
+  vaadin-vertical-layout {
```

Comment:
I don't see vaadin-ordered-layout dependency pulled in anywhere?

Are you sure you've updated yarn.lock for this PR?

Reply:
I don't see any dependency for `vaadin-ordered-layout` in my local `yarn.lock`.
` "@vaadin/vaadin-ordered-layout": "^1.3.0" ` is only listed in `devDependencies` for `vaadin-accordion` module itself.

Reply:
> "@vaadin/vaadin-ordered-layout": "^1.3.0" is only listed in devDependencies for vaadin-accordion module itself.

Can you send a link to package.json? I don't see such anywhere in https://github.com/vaadin/vaadin-accordion/blob/master/package.json

Also, your own yarn.lock in this PR https://github.com/conversionxl/aybolit/blob/5b65345499d649446fd928248945276b486dfd6a/yarn.lock doesn't contain vaadin-ordered-layout anywhere, so what is this `vaadin-vertical-layout` tag you're styling here?